app/models/zotero_api/bibliography.rb

```diff
@@ -0,0 +1,21 @@
+module ZoteroApi
+  # A complete Zotero bibliography
+  class Bibliography < Array
+    # @return [Bibliography] all items sorted by Author-Date
+    def sort_by_author_date
+      self.class.new(sort_by(&:to_author_date))
+    end
+
+    # @param [String] `key` the bibliography item's `key` value
+    # @return [BibliographyItem] the matching item, or `nil`
+    def find(key)
```

Comment:
Doesn't this mask `Enumerable#find`? (and, the `each` below it looks suspiciously like it should use that..)